### ISSUE TITLE
acme: uci: Parse error (invalid command) at line 23, byte 1

### ISSUE THREAD

**Issue by alex9434** [openwrt/packages#24074]
Maintainer: @hgl  @tohojo @neheb @httpstorm @scde @lialosiu 
Environment: OpenWrt 23.05.3

Description:
uci shows configuration error in file
```
for CONF in /etc/config/* ; do uci show "${CONF##*/}" > /dev/null || echo "${CONF}"; done
uci: Parse error (invalid command) at line 23, byte 1
/etc/config/acme
```

**Comment by stangri** [comment]
There was a PR with the fix, I don't understand why it was closed: https://github.com/openwrt/packages/pull/23615

**Comment by tohojo** [comment]
Because this was already fixed with acme-common v1.0.4 - please update to that version
